PlaybackLab/ControlsLockScreenSampleApp/ControlsLockScreenSampleApp/Views/ListTableViewController.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  ListTableViewController.swift
+//  ControlsLockScreenSampleApp
+//
+//  Created by Carlos Ceja on 10/18/17.
+//  Copyright © 2017 Ooyala. All rights reserved.
+//
+
+import UIKit
+
+class ListTableViewController: UITableViewController {
+  
+  private let options: [PlayerSelectionOption] = OptionDataSource.options()
+  
+  private var cellIdentifier: String = "cellOption"
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    tableView.tableFooterView = UIView()
+    tableView.bounces = false
+  }
+  
+  override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
```

Comment:
Both `supportedInterfaceOrientations` and `shouldAutorotate` seem to not be working. The table still rotates to landscape mode.

Reply:
I needed to override other methods. I decided to remove, we don't need it for this sample app.